packages/components/flex/docs.mdx

```diff
@@ -51,7 +51,9 @@ The HTML DOM structure is
   </Flex>
 </Playground>
 
-## Usage
+___
+
+## Flex.Column
```

Comment:
Would prefer to have title as dot notation and display all dot notations together as example